### PR TITLE
Android sign-out guide

### PR DESCRIPTION
**Pull request by FeiChen-okta**
## Description:

Android sign-out guide.
Added a section sign-out-page. Not sure if we want to put this with sign-in-page.

### Resolves:

* [OKTA-220642](https://oktainc.atlassian.net/browse/OKTA-220642)

### CODE REVIEW

**Comment by FeiChen-okta** [comment]
Updated with changes.

File: packages/@okta/vuepress-site/guides/sign-into-mobile-app/sign-out-page/android/signout.md [hunk 1]
```diff
@@ -0,0 +1,46 @@
...
+            //handle result
+        }
+        @Override
+        public void onError(String error, AuthorizationException exception) {
+            //handle request error
+        }
+    });
+```
+
+**Note:** *Access, refresh, and ID tokens need to be revoked in separate requests.*
```
Comment (susanharper-okta) [inline packages/@okta/vuepress-site/guides/sign-into-mobile-app/sign-out-page/android/signout.md]: sorry - didn't notice this before - but the note should be enough to call out the information. The italics can be removed. 

**Note:** Access, refresh, and ID tokens need to be revoked in separate requests.

File: packages/@okta/vuepress-site/guides/sign-into-mobile-app/sign-out-page/index.md [hunk 2]
```diff
@@ -0,0 +1,13 @@
+---
+title: Open the Sign-Out Page
+---
+To sign out users, open a web view and display the hosted sign-out page.
+
+<StackSelector snippet="signout"/>
+
+Signing out signs the user out if the subject in the ID token matches the current Okta session. A `post_logout_redirect_uri` may be specified to redirect the browser after the sign out is performed, for example, `com.oktaorg.name:/logout`. Otherwise, the browser is redirected to the Okta sign-in page.
```
Comment (susanharper-okta) [inline packages/@okta/vuepress-site/guides/sign-into-mobile-app/sign-out-page/index.md]: One small change with this first sentence:

Users are signed out if the subject in the ID token matches the current Okta session.

**Comment by FeiChen-okta** [comment]
Updated with changes.

**Pull request merged**